### ISSUE TITLE
Apps may not schedule more than 100 distinct jobs

### ISSUE THREAD

**Issue by nlgtuankiet** [blueshift-labs/Blueshift-Android-SDK#103]
Recently we encountered a large number of crashes when using the `Blueshift-Android-SDK`

```
Caused by java.lang.IllegalStateException: Apps may not schedule more than 100 distinct jobs
       at android.os.Parcel.readException(Parcel.java:1691)
       at android.os.Parcel.readException(Parcel.java:1636)
       at android.app.job.IJobScheduler$Stub$Proxy.schedule(IJobScheduler.java:158)
       at android.app.JobSchedulerImpl.schedule(JobSchedulerImpl.java:42)
       at com.blueshift.request_queue.RequestQueue.scheduleQueueSyncJob(RequestQueue.java:60)
       at com.blueshift.Blueshift.initialize(Blueshift.java:288)
```

Ideas that might improve the situation:
1. wrap a try-catch block to `com.blueshift.Blueshift#initialize` 
2. use `android.app.job.JobScheduler#cancel` to cancel `com.blueshift.model.Configuration#networkChangeListenerJobId` and `com.blueshift.model.Configuration#bulkEventsJobId` before calling `com.blueshift.Blueshift#initialize`

Any solutions to this?

**Comment by rahulrvp** [comment]
Hey @nlgtuankiet thanks for reporting the issue. We'll take a look at this. Could you please raise a support ticket from your work email in the meantime?

**Comment by rahulrvp** [comment]
We have fixed this issue in v3.0.1. Please update the Blueshift Android SDK.
I am going ahead and closing this issue now. Please feel free to reach out if you need any help.

Cheers!
